src/codegen/codegen_neuron_cpp_visitor.hpp

```diff
@@ -784,12 +784,17 @@ class CodegenNeuronCppVisitor: public CodegenCppVisitor {
      */
     void print_cvode_definitions();
 
+    /* print the CVODE function returning the # of ODEs to solve */
     void print_cvode_count();
 
+    /* print the CVODE function for setup of tolerances */
     void print_cvode_tolerances();
 
-    void print_cvode_update(const std::string& name, const ast::StatementBlock& node);
+    /* print the CVODE update function ``name`` from ``block`` */
```

Comment:
The rest of the code uses either `\c name` or single backticks, e.g. \`name\`. This looks like what one would do in Sphinx/RST.